Samples/PipeableSample/PipeableSample/PipeableWebViewWrapper.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import PipeableSDK
+import SwiftUI
+import WebKit
+
+struct PipeableWebViewWrapper: UIViewRepresentable {
+    @Binding var items: [NewsItem]
+    var onClose: () -> Void
+
+    func makeUIView(context _: Context) -> PipeableWebView {
+        let webView = PipeableWebView()
+        let page = webView.page
+
+        Task {
+            _ = try? await page.goto("https://news.ycombinator.com", waitUntil: .networkidle)
+
+            try? await sleep(ms: 1000)
+
+            let searchForm = try await page.waitForXPath(
+                "//form[contains(@action, 'hn.algolia.com')]",
+                visible: true
+            )
+
+            guard let textAreaEl = try await searchForm?.querySelector("input[type='text']") else {
+                return
+            }
+
+            _ = try await page.evaluateAsyncFunction(
+                """
+                    el.scrollIntoView();
+
+                    el.focus();
+
+                    await new Promise((res) => setTimeout(res, 500));
+
+                    // Scroll a bit to the top, so we can visualize what we are typing.
+
+                    window.scrollTo(0, window.scrollY + 300);
+                """, arguments: ["el": textAreaEl]
+            )
+
+            try? await sleep(ms: 1000)
+
+            try await textAreaEl.type("web automation", delay: 50)
+
+            _ = try await page.submitActiveForm()
+
+            try await page.waitForURL { url in url.contains("hn.algolia") }
+
+            _ = try await page.waitForSelector(".Story_title", visible: true)
+
+            // Hold a bit, so it's visible.
+            try? await sleep(ms: 2000)
+
+            let stories = try await page.querySelectorAll(".Story_title a > span")
+
+            items = []
+
+            var storyTitles: [String] = []
+
+            for story in stories {
+                if let title = try await story.textContent() {
+                    items.append(NewsItem(itemName: title))
+                    storyTitles.append(title)
+                }
+            }
+
+            print(storyTitles)
+
+            onClose()
+        }
+
+        return webView
+    }
+
+    func updateUIView(_: PipeableWebView, context _: Context) {}
```

Comment:
minor: can we remove this or add a comment explaining why we left it?

Reply:
We can't remove it, need to implement the UIViewRepresentable protocol, which requires this. I can add a comment.